sci-libs/ondselsolver/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <!-- maintainer-needed -->
```

Comment:
We can't add maintainer needed packages. Would you be willing to act as the maintainer for this package?

Reply:
I would like to do it because I use Assemby, but firs I need to pass through the process of becoming a developer with right to commit. It'll take some time before accomplish it, meanwhile could we upload the ondselsolver to GURU and enable Assembly flag into gentoo freecad ebuild?

Reply:
An ebuild in ::gentoo cannot depend on an ebuild in ::guru. But you could maintain this in ::gentoo via proxy if you want.

Reply:
I want to, I'm reading the instructions to be a proxy maintainer, I've already made some simpler ebuilds. Which would be the best approach to start with this package?

Reply:
> Which would be the best approach to start with this package?

Great! I propose to start with cloning this PR and adding yourself here in the `metadata.xml` and then opening a new PR.

Reply:
I'm trying to create a PR from gh client, I'm currently learning to use it:
`gh pr create --base ondselsolver-20240909 --head master --title "Ondselsolver new proxy maintainer" --body "..."`
I get the following message: 
`Creating pull request for master into ondselsolver-20240909 in cjmayo/gentoo`
`pull request create failed: GraphQL: must be a collaborator (createPullRequest)`
Does I need PR permission?

Reply:
I' ve created PR #39744